.eslintignore

```diff
@@ -28,6 +28,8 @@ packages/gatsby-admin/public
 packages/gatsby/gatsby-admin-public
 packages/gatsby-codemods/transforms
 packages/gatsby-source-graphql/batching
+packages/gatsby-plugin-gatsby-cloud
+packages/gatsby-plugin-preact
```

Comment:
This will disable linting completely. We do want to lint source code, just not built/transpiled code.  There doesn't seem to be nice way to say "Ignore most of `packages/gatsby-plugin-gatsby-cloud` but not `packages/gatsby-plugin-gatsby-cloud/src`" (they say there is with `!` negation, but I didn't get it to work as I would expect it), so I would suggest ignoring individual dirs of built files, so that we can keep linting on `src` directory:
```suggestion
packages/gatsby-plugin-gatsby-cloud/components
packages/gatsby-plugin-gatsby-cloud/utils
packages/gatsby-plugin-preact/fast-refresh
```

Reply:
Yep that makes much more sense - thanks @pieh